### DISCUSSION TITLE
Problems with Data and Ethics: Palantir

### DISCUSSION THREAD

**Discussion by vcta99** [albertkun/21S-ASIAAM-191A#40] [Past Thinking Caps]
I remember this being a huge topic ~2018 within the CS community at UCLA, but I wanted to spotlight Palantir and the use of unethical data analytics to target undocumented communities. Palantir was developed as a post 9/11 data analytics corporation in order to ensure that "the West, especially America, the strongest in the world, for the sake of global peace and prosperity." (Palantir CEO Alex Karp). It is easy to see how Palantir can further marginalize vulnerable communities under the guise of patriotism. Although I am not tech-savvy enough to fully understand Palantir's process, I understand that the company uses data mining to target specific demographics, and that it was initially used to determine familial relations between undocumented immigrants, their children, and their sponsors. This lead to one of the largest deportation raids in US history. In the Guardian article listed below, it's state that these raids were conducted by using Palantir software in the phones of those who were affected.

To me, that is the biggest problem regarding ethics in data analytics. There is a lack of consent and awareness on how our data is being used and manipulated against us -- especially in regards to more vulnerable populations who are not as technologically literate. What's more concerning is that I am not quite sure what the solution is. How can we empower users/make technology more consentful?

https://www.theguardian.com/commentisfree/2020/sep/04/palantir-ipo-ice-immigration-trump-administration

**Comment by gu-ana** [comment]
This came to mind when I was thinking about misuse of data as well. When I had went to a career fair with a representative from Palantir, they basically skirted around the question(I mean who wouldn't when there isn't a PR representative) when asked about contracts with US government agency ICE. It definitely has made me avoid applying to any of their jobs.

**Comment by ninalong** [comment]
Thank you for sharing this, this was something I didn't know about. I wonder how the CS community, who I assume isn't particularly political most of the time (my assumption as a North Campus major HAHA) responded to this. I think this is a really important juncture of policy and tech, with a vital imperative to impose greater regulations on tech companies.

**Comment by lvalles10** [comment]
I really appreciate you sharing this perspective; this also brings to mind the way AI and data is manipulated in China to identify Uighur Muslims and forcibly input them into re-education camps (https://www.nytimes.com/2019/04/14/technology/china-surveillance-artificial-intelligence-racial-profiling.html). In addition to private companies, it's dangerous what governments can do with weaponized data, especially to already marginalized and oppressed groups.
